Doc how to install citrus jbang

Can we make a specific doc like we have for jbang install (install.md) that we can link to for citrus as well
https://github.com/apache/camel-jbang-examples/tree/main/mqtt#integration-testing

Also the citrus docs seems wrong
```
~/workspace/camel-jbang-examples main ❯ jbang install citrus@citrusframework/citrus
[jbang] [ERROR] Script or alias could not be found or read: 'install'
[jbang] Run with --verbose for more details. The --verbose must be placed before the jbang command. I.e. jbang --verbose run [...]
~/workspace/camel-jbang-examples main ❯ jbang app install citrus@citrusframework/citrus
```

--- a/README.adoc
+++ b/README.adoc
@@ -55,7 +55,7 @@ You need to install Citrus as a JBang app, too:
 jbang app install citrus@citrusframework/citrus
 ----
 
-Now you can start running commands for the Citrus JBang app with just calling `citrus`:
+Now you can start running commands for the Citrus JBang app with `citrus`:
 
 [source,shell]
 ----
@@ -69,6 +69,7 @@ You can run the test with Citrus JBang like this:
 
 [source,shell]
 ----
+cd test
 citrus run mqtt.camel.it.yaml
 ----
 

--- a/ftp/README.adoc
+++ b/ftp/README.adoc
@@ -74,12 +74,13 @@ Then you can browse: http://localhost:8080/q/dev to introspect the running Camel
 == Integration testing
 
 The example provides an automated integration test (`ftp.camel.it.yaml`) that you can run with the https://citrusframework.org/[Citrus] test framework.
-Please make sure to install Citrus as a JBang app, too.
+Please make sure to install Citrus as a JBang application (see link:../../install-citrus.adoc[Citrus installation guide]).
 
 You may run the test with:
 
 [source,shell]
 ----
+cd test
 citrus run ftp.camel.it.yaml
 ----
 

--- a/install-citrus.adoc
+++ b/install-citrus.adoc
@@ -1,0 +1,20 @@
+First install JBang according to the https://www.jbang.dev/download/[JBang installation guide]
+
+For a more convenient way to use the Citrus JBang scripts you can install Citrus as a JBang app.
+
+.Install Citrus app
+[source,shell]
+----
+jbang trust add https://github.com/citrusframework/citrus/
+jbang app install citrus@citrusframework/citrus
+----
+
+Now you can run Citrus with `citrus` to execute some tests:
+
+.Run my.camel.it.yaml
+[source,shell]
+----
+citrus run my.camel.it.yaml
+----
+
+For more information about using Citrus JBang please refer to

--- a/install.adoc
+++ b/install.adoc
@@ -1,6 +1,6 @@
-First install JBang according to https://www.jbang.dev
+First install JBang according to the https://www.jbang.dev/download/[JBang installation guide]
 
-When JBang is installed then you should be able to run from a shell:
+When JBang is installed you should be able to run from a shell:
 
 [source,shell]
 ----

--- a/mqtt/README.adoc
+++ b/mqtt/README.adoc
@@ -81,12 +81,13 @@ camel stop mqtt
 == Integration testing
 
 The example provides an automated integration test (`mqtt.camel.it.yaml`) that you can run with the https://citrusframework.org/[Citrus] test framework.
-Please make sure to have installed Citrus as a JBang application.
+Please make sure to install Citrus as a JBang application (see link:../../install-citrus.adoc[Citrus installation guide]).
 
 You can run the test with:
 
 [source,shell]
 ----
+cd test
 citrus run mqtt.camel.it.yaml
 ----
 

